packages/connect/src/index.js

```diff
@@ -1,5 +1,5 @@
 export { observe, unobserve } from "./observer";
-export { observable, isObservable, raw } from "./observable";
+export { observable, isObservable, raw, clone } from "./observable";
```

Comment:
Could you please move `clone` to `tiny-router`? Sorry I think I probably told you to put it here but I don't want this to be a public API of `connect` yet. I want to discuss it with Miklos and see if how this can be avoided.